auth-internal/src/test/java/com/sequenceiq/cloudbreak/auth/security/internal/InternalCrnModifierTest.java

```diff
@@ -24,6 +24,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
+import com.sequenceiq.cloudbreak.auth.ReflectionUtil;
```

Comment:
Do we really need this?

Reply:
yes, it is used in the test, i just moved that class into common, so we have to import it